Source/Core/DiscIO/VolumeWiiCrypted.cpp

```diff
@@ -160,10 +160,43 @@ IVolume::ECountry CVolumeWiiCrypted::GetCountry() const
 	if (!m_pReader)
 		return COUNTRY_UNKNOWN;
 
+	u32 region_code;
+	m_pReader->Read(0x4E000, sizeof(u32), (u8*)&region_code);
+	region_code = Common::swap32(region_code);
 	u8 country_code;
 	m_pReader->Read(3, 1, &country_code);
 
-	return CountrySwitch(country_code);
+	IVolume::ECountry rv = CountrySwitch(country_code);
+	switch (region_code)
+	{
+		case 0:
+			switch (rv) {
+				case IVolume::COUNTRY_TAIWAN:
+				case IVolume::COUNTRY_JAPAN:
+					return rv;
+			}
+			return IVolume::COUNTRY_JAPAN;
```

Comment:
Is there a good reason, not to put this in the default case of the switch statement?